docs/intelligentapps/finetune.md

```diff
@@ -0,0 +1,278 @@
+---
+Order: 5
+Area: intelligentapps
+TOCTitle: Finetune
+ContentId:
+PageTitle: Finetune AI Models
+DateApproved:
+MetaDescription: Use custom dataset to finetune a generative AI model in the Azure cloud or locally with GPUs. Deploy the finetuned model to the Azure cloud or download adapter file.
+MetaSocialImage:
+---
+
+# Finetune models
+
+# **[Preview]** Introducing Remote Development with AI Toolkit for VS Code
```

Comment:
Don't use H1 except for the title.

Why is remote development part of an article about finetuning?

Reply:
a little confusing, so had this rewritten.